pkg/ebpf/tracee.go

```diff
@@ -572,6 +572,24 @@ func (t *Tracee) initDerivationTable() error {
 				),
 			},
 		},
+		events.SchedProcessExec: {
+			events.ProcessCreation: {
+				Enabled:        t.events[events.ProcessCreation].submit,
+				DeriveFunction: derive.ProcessCreation(),
```

Comment:
It might be confusing to call exec operation process creation (which is actually done by fork/clone).
I suggest changing this to ProcessExec